src/Patat/Presentation/Interactive.hs

```diff
@@ -32,7 +33,7 @@ data PresentationCommand
 --------------------------------------------------------------------------------
 readPresentationCommand :: IO PresentationCommand
 readPresentationCommand = do
-    k <- readKey
+    k <- GetKey.getKey
     case k of
         "q"      -> return Exit
         "\n"     -> return Forward
```

Comment:
I tried applying these changes over the most recent version in hopes of getting it to work on Windows; testing this out on the first 01.md, it didn't appear to do anything differently. Advancing worked fine, but it was impossible to move backwards on slides.

However, commenting out this particular line and freeing `\n` seems to resolve some kind of fight between the windows console and getChar, allowing you to write a command and then flush the buffer with enter, making patat usable for me on W10 1809.